data.json

```diff
@@ -0,0 +1,39 @@
+[
```

Comment:
It is unfortunate and I've already felt a taste of it. However, it saves us from weird loading problems on state-side schema changes (which has actually already happened). Essentially the workflow is to download the file, convert it to geojson and check its fields against those that the loader expects. If there is no difference, we can assume that there were just edits/additions and update the hash. If there *are* changes to the schema, the transformer in the loader (and associated tests) need to be updated).

Currently I've been doing this by hand, but I will automate it as soon as I feel confident that I can account for edge cases.